fit/core-reference/src/main/resources/log.properties

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Hum, that'd be even worse, and I don't think that the proposed change would resolve to the same file...